torchvision/csrc/cpu/image/image_read_mode.h

```diff
@@ -0,0 +1,9 @@
+#pragma once
+
+/* Should be kept in-sync with Python ImageReadMode enum */
```

Comment:
I thought it would be a good idea to have constants on C++ instead on relying on integers. Originally I was going for proper enums but I believe this is not allowed by the dispatcher. The caveat of having any declaration in C++ is that we will have to keep them in-synch with the values in Python.

Reply:
Yep, this sounds good to me, although having to sync them is a bit annoying, I don't think we can do better than that for now